tests/dummy/app/resolver.js

```diff
@@ -1,3 +1,12 @@
 import Resolver from 'ember-resolver';
 
-export default Resolver;
+export let registry = Object.create(null);
+export function setRegistry(newRegistry) {
+  registry = newRegistry;
+}
+
+export default Resolver.extend({
+  resolve(fullName) {
+    return registry[fullName] || this._super(...arguments);
+  },
+});
```

Comment:
is that going to be required for all apps?

Reply:
No, absolutely not!  This is _only_ a thing needed to test things internally here.  Basically the `setResolverRegistry` test helper function (just here in the tests of this repo) resets the list of "known" modules so that we can test things like `this.owner.register` "wins" over what is resolved, and whatnot.